warehouse/query-core/src/test/java/datawave/query/rules/FieldExistenceRuleTest.java

```diff
@@ -95,4 +106,12 @@ protected ShardQueryRule getNewRule() {
         rule.setSpecialFields(fieldExceptions);
         return rule;
     }
+
```

Comment:
Rather than use an AccumuloClient here, it would be simpler to instead mock out the expected interactions with the helper as you did in PR #3174. This way, the FieldExistenceRuleTest does not need to care about the implementation details of the MetadataHelper.